GitCommands/Worktrees/WorktreeInfoResult.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+
+namespace GitCommands.Worktrees
+{
+    public class WorktreeInfoResult
+    {
+        public WorktreeInfoResult(IList<WorkTreeInfo> worktrees, WorkTreeInfo? currentWorktree)
+        {
+            var trees = new List<WorkTreeInfo>();
```

Comment:
Avoid `var` where easily possible in general:
```suggestion
            List<WorkTreeInfo> trees = new();
```